### PR TITLE
completed react-function-components exercise - Nick Sturz

### PR DESCRIPTION
**Pull request by nsturz**
https://user-images.githubusercontent.com/94485412/187793394-b2391aee-a00b-4cb3-960c-9db0218e191b.mov

### CODE REVIEW

File: react-function-components/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+function CustomButton(props) {
+  return <button>Click Me!</button>;
+}
+
+const container = document.getElementById('root');
+
+const root = ReactDOM.createRoot(container);
+
+root.render(CustomButton());
```
Comment (scbowler) [inline react-function-components/src/index.jsx:12]: With components we never directly call them. Instead we write them as JSX:
```JSX
root.render(<CustomButton />);
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Heading in the right direction, but one update needed. Let me know if you have any questions.

File: react-function-components/src/index.jsx (hunk 1 above)
Comment (nsturz) [reply react-function-components/src/index.jsx]: ah good to know! addressed. definitely missed that.

File: react-function-components/src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```
Comment (scbowler) [inline react-function-components/src/index.jsx:2]: Here you need to specify `client`.
```JSX
import ReactDOM from 'react-dom/client';
```

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍 

1 more FYI comment, missed it the first time, but no changes required.

**Pull request merged**